test/stratonovich_convergence_tests.jl

```diff
@@ -32,6 +32,9 @@ sim  = test_convergence(dts,prob,ImplicitRKMil(interpretation=:Stratonovich),num
 sim  = test_convergence(dts,prob,SROCK1(interpretation=:Stratonovich),numMonte=Int(2e2))
 @test abs(sim.𝒪est[:l2]-1) < 0.15
 
+sim  = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(2e2))
+@test abs(sim.𝒪est[:final]-2) < 0.20
```

Comment:
this is strong convergence?

Reply:
This is a linear problem that’s why the approx 2.0 order convergence.